src/exceptions.jl

```diff
@@ -0,0 +1,7 @@
+struct LayerNotFound <: Exception
```

Comment:
What do you think of:

```suggestion
struct LayerNotFoundException <: Exception
```